src/test/java/org/prebid/server/bidder/ttx/TtxBidderTest.java

```diff
@@ -60,8 +65,10 @@ public void makeHttpRequestsShouldReturnErrorIfImpExtCouldNotBeParsed() {
 
         // then
         assertThat(result.getErrors()).hasSize(1);
-        assertThat(result.getErrors().get(0).getMessage()).startsWith("Cannot deserialize instance");
-        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).allMatch(error -> error.getMessage()
+                .startsWith("Cannot deserialize instance")
+                && error.getType() == BidderError.Type.bad_input);
+        assertThat(result.getValue()).hasSize(1);
     }
 
     @Test
```

Comment:
makeHttpRequestsShouldSetSiteIdFromImpExt will be updated if you add null check for site